cirrus/google_cloud/manager.py

```diff
@@ -917,6 +934,94 @@ def delete_service_account_key(self, account, key_name):
 
         return response.json()
 
+    def give_service_account_billing_access(
+        self, account_id, project_id=None, billing_role_name=None
+    ):
+        """
+        Give the specified service account id permissions to bill to the provided project
+        id. Project id will default to the current project.
+
+        Args:
+            account_id (str): email address or the uniqueId of the service account
+            project_id (str): Google project identifier
+        """
```

Comment:
you're missing `billing_role_name` in the docstring
and what is the use case for providing a custom name?

Reply:
just in case someone has a preference as to the role name in the future, ends up being an API call change in fence instead of another release in cirrus. :man_shrugging: not a big deal, could just leave it hard-coded for now